demos/blog/protected/views/layouts/main.php

```diff
@@ -1,28 +1,30 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<!DOCTYPE html>
+<html lang="en">
 <head>
-	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<base href="<?=Yii::app()->baseUrl; ?>/">
+	<meta charset="<?=Yii::app()->charset; ?>" />
 	<meta name="language" content="en" />
 
 	<!-- blueprint CSS framework -->
-	<link rel="stylesheet" type="text/css" href="<?php echo Yii::app()->request->baseUrl; ?>/css/screen.css" media="screen, projection" />
-	<link rel="stylesheet" type="text/css" href="<?php echo Yii::app()->request->baseUrl; ?>/css/print.css" media="print" />
+	<?=CHtml::cssFile('css/screen.css', 'screen, projection');?>	
+	<?=CHtml::cssFile('css/print.css', 'print');?>	
+	
 	<!--[if lt IE 8]>
-	<link rel="stylesheet" type="text/css" href="<?php echo Yii::app()->request->baseUrl; ?>/css/ie.css" media="screen, projection" />
+	<?=CHtml::cssFile('css/ie.css', 'screen, projection');?>	
 	<![endif]-->
 
-	<link rel="stylesheet" type="text/css" href="<?php echo Yii::app()->request->baseUrl; ?>/css/main.css" />
-	<link rel="stylesheet" type="text/css" href="<?php echo Yii::app()->request->baseUrl; ?>/css/form.css" />
+	<?=CHtml::cssFile('css/main.css');?>	
+	<?=CHtml::cssFile('css/form.css');?>	
 
-	<title><?php echo CHtml::encode($this->pageTitle); ?></title>
+	<title><?=Yii::app()->name;?></title>
```

Comment:
That's not a good idea:

1. We need page title, not application name.
2. Missing `CHtml::encode()` introduced XSS.

Reply:
Got it, I got it wrong, thank you.